pennylane/labs/resource_estimation/ops/op_math/controlled_ops.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Resource operators for controlled operations."""
+from typing import Dict
+
+import pennylane as qml
+import pennylane.labs.resource_estimation as re
+
+# pylint: disable=arguments-differ,too-many-ancestors
+
+
+class ResourceControlledPhaseShift(qml.ControlledPhaseShift, re.ResourceOperator):
+    r"""Resource class for the ControlledPhaseShift gate.
+
+        Resources:
```

Comment:
The title "Resources:" looks a bit strange in the docs, please consider removing  this line in all classes.

Reply:
I think the heading should be fine to keep. It should  be a separate section from the traditional description of the operator. It renders well for me, is this what you are seeing?

<img width="796" alt="image" src="https://github.com/user-attachments/assets/67e415f6-3e7f-4ec7-aa8b-306c7da3ac0e">